src/lib/update-alternatives/UI/alternatives_dialog.rb

```diff
@@ -66,8 +58,10 @@ def auto_handler
     end
 
     def alternatives_handler
-      selected_alternative = Yast::UI.QueryWidget(Id(:alternatives), :CurrentItem)
-      Yast::UI.ChangeWidget(Id(:slaves), :Value, @mock_slaves[selected_alternative])
+      selected_choice = Yast::UI.QueryWidget(Id(:alternatives), :CurrentItem)
+      choice = @alternative.choices.select { |e| e.path == selected_choice }.first
+      slaves = "<pre>" + choice.slaves + "</pre>"
+      Yast::UI.ChangeWidget(Id(:slaves), :Value, slaves)
```

Comment:
hmm, how it act if it is then changed? You query currentItem above, so it there is more changes I expect it sneak into <pre> one, right?